build-docker.sh

```diff
@@ -3,11 +3,40 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 BUILD_OPTS="$*"
 
-DOCKER="docker"
-
-if ! ${DOCKER} ps >/dev/null 2>&1; then
-	DOCKER="sudo docker"
+SUDO=''
+if [ $EUID != 0 ]; then
+    SUDO='sudo'
 fi
+
+function install_binfmt_support {
+    local binfmtSupport="binfmt-support"
+    if ! dpkg -s $binfmtSupport &>/dev/null; then
```

Comment:
Since the docker build is especially made to be run on other distros like Fedora & CentOS this should be taken into consideration here. So please also support `rpm` or ignore OSes other than Ubuntu & Debian.